support-frontend/assets/helpers/rendering/render.ts

```diff
@@ -46,19 +46,20 @@ const renderError = (e: Error): void => {
 	);
 };
 
-const renderPage = (
-	content: React.ReactElement<React.DOMAttributes<Element>>,
-): void => {
+const renderPage = (content: ReactElement<DOMAttributes<Element>>): void => {
 	const element: HTMLElement | null = document.querySelector('.gu-render-to');
 
 	if (element) {
 		delete element.dataset.notHydrated;
 
 		try {
 			if (process.env.NODE_ENV === 'development' && !isSafari) {
-				// @ts-expect-error - Not sure why it's not finding typedefs for Preact
 				void import('preact/debug');
-				void import('@axe-core/react').then((axe) => {
+				void Promise.all([
```

Comment:
@jamesgorrie are you able to run the site in Dev mode locally? I'm seeing an issue in the client-side bundle that looks like it's related to the updates in this file.

<img width="571" alt="Screenshot 2024-06-06 at 17 31 56" src="https://github.com/guardian/support-frontend/assets/1590704/dcf794c0-89dc-401b-bac2-795dc1faf3fb">

Reply:
Hmm - yea, I've removed that refactor as it's not actually part of this and didn't reduce the package size either...